doc/QUICK_START.md

```diff
@@ -34,10 +34,44 @@ Detailed installation instructions for different platforms are located on the [O
 
 You can find information on configuring OpenProject in [`config/CONFIGURATION.md`](CONFIGURATION.md).
 
-## Fast install
-
 These are generic (and condensed) installation instructions for the **current dev** branch *without plugins*, and optimised for a development environment. Refer to the OpenProject website for instructions for the **stable** branch, OpenProject configurations with plugins, as well as platform-specific guides.
 
+## Fast install (Docker version)
+
+### Prerequisites
+
+* Git
+* [Docker](http://www.docker.com/)
+* [Docker Compose](https://docs.docker.com/compose/)
+
+To install Docker you can follow the [Get Started with Docker](https://docs.docker.com/mac/) guide.
```

Comment:
I think you should reference https://docs.docker.com/engine/installation/ instead.